addons/mrp_subcontracting_dropshipping/tests/test_anglo_saxon_valuation.py

```diff
@@ -207,6 +209,7 @@ def test_avco_valuation_subcontract_and_dropshipped_and_backorder(self):
             ]
         )
 
+    @freeze_time('2024-03-13')
```

Comment:
I don't like freezetime for this kind of purpose since it can break for another reason in the future. (You cannot validate something more than 2 years old .... already seen and some test are breaking this year for similar reasons)

Wouldn't it be simple to modify the assertion in the bottom to use values from the account_move instead of hardcoded 2024?

Reply:
ok, I've modified it if it's better 👍

Reply:
Thanks, perfect!